src/fidesops/service/connectors/query_config.py

```diff
@@ -121,17 +131,56 @@ def update_value_map(self, row: Row, policy: Policy) -> Dict[str, Any]:
         for a given customer_id will be replaced with null values.
 
         """
-        rule_to_collection_fields = self.build_rule_target_fields(policy)
+        rule_to_collection_fields: Dict[
+            Rule, List[str]
+        ] = self.build_rule_target_fields(policy)
 
         value_map: Dict[str, Any] = {}
         for rule, field_names in rule_to_collection_fields.items():
             strategy_config = rule.masking_strategy
-            strategy = get_strategy(
```

Comment:
What about on #139: `for rule, field_names in rule_to_collection_fields.items() if rule.masking_strategy:` to get rid of the extra conditional on #141?

Reply:
perfect thanks!